web/src/containers/ActivityDetailContractorResources.js

```diff
@@ -26,7 +27,7 @@ class ActivityDetailContractorExpenses extends Component {
     const { activity, updateActivity } = this.props;
 
     const updates = {
-      contractorResources: { [index]: { years: { [year]: +value } } }
+      contractorResources: { [index]: { years: { [year]: value } } }
```

Comment:
I'm not wild about this.  The text mask input sends a formatted string (e.g., `$1,000.00`) with the `onChange` event.  That's problematic for two reasons:

1. if we add that formatted string to our internal state, we can't easily use it for math later on
2. we want to store numeric data on the API for the same reason

This change will actually break #544 _right now_, so it's not a future worry.  Would it be sensible to further wrap the text mask with specific `<DollarInput>` and `<PercentInput>` components that de-format the text and send numbers in their `onChange` events?

Reply:
yes, i hear you and agree -- i feel like this library should really handle this (returning the raw and formatted values) -- maybe it does, i'll check!

Reply:
alright, see what you think!